src/main/java/net/william278/velocitab/Velocitab.java

```diff
@@ -170,6 +173,14 @@ private void disableScoreboardManager() {
         }
     }
 
+    private void prepareSortingManager() {
+        this.sortingManager = new SortingManager(this);
+    }
+
+    public SortingManager getSortingManager() {
```

Comment:
Either annotate as @NotNull or make it return an optional (as per previous comment)

Reply:
I would go with NotNull